.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
newline

(https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)